packages/renderer/index.d.ts

```diff
@@ -419,11 +416,11 @@ declare namespace ReactPDF {
 
   interface RadialGradientProps {
     id: string;
-    cx?: string | number;
-    cy?: string | number;
-    fr?: string | number;
-    fx?: string | number;
-    fy?: string | number;
+    cx: string | number;
+    cy: string | number;
+    fr: string | number;
+    fx: string | number;
+    fy: string | number;
```

Comment:
Ditto

Reply:
yeah, cleaning up & adding tests, going to force push over this